Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/setup.py

```diff
@@ -544,7 +545,9 @@ def main(cls, args, repository, **kwargs):
                 info_url = 'https://github.com/WebKit/WebKit/wiki/Git-Config#Configuration-Options'
                 print('For detailed information about the options configured by this script, please see:\n{}'.format(info_url))
                 if not args.defaults and Terminal.choose("Would you like to open this URL in your browser?", default='Yes') == 'Yes':
-                    if not Terminal.open_url(info_url):
```

Comment:
The point of `Terminal.open_url` is that code like this can be insulated from decisions we make about how URLs should be opened. If we were going to make this change, it would be in `terminal.py`, not here here.

Reply:
This brings us back to the original problem of Windows opening an explorer tab, not a web browser